ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -3143,7 +3201,7 @@ mod tests {
     /// Ensure that any given instruction fits in 64-bits.
     #[test]
     fn inst_size() {
-        assert!(mem::size_of::<Inst>() <= mem::size_of::<u64>());
```

Comment:
Docstring above needs updating.

Reply:
Me may as well say our instructions are `u128` sized rather than "two 64-bit sized"

Reply:
Fixed in the force push.